lib/chrome-driver.js

```diff
@@ -24,7 +23,7 @@ ChromeDriver.prototype.start = function () {
     cwd: process.cwd(),
     env: this.getEnvironment()
   }
-  this.process = ChildProcess.spawn(process.execPath, args, options)
+  this.process = ChildProcess.spawn(this.path, args, options)
```

Comment:
You would need to set it here I believe: https://github.com/kevinsawicki/spectron/blob/93aa264421252bf229ca91b3d63a0bd1475a6b41/lib/chrome-driver.js#L34

Perhaps something like:

```js
if (process.versions.electron) {
  env.ATOM_SHELL_INTERNAL_RUN_AS_NODE = 1
}
```